README.md

```diff
@@ -17,13 +17,11 @@ cluster on top of bare metal infrastructure using Metal3.
 
 ## Compatibility with Cluster API
 
-| CAPM3 version | Cluster API version | CAPM3 Release |
-|---------------|---------------------|---------------|
-| ~~v1alpha4~~  | ~~v1alpha3~~        | ~~v0.4.X~~    |
-| v1alpha5      | v1alpha4            | v0.5.X        |
-| v1beta1       | v1beta1             | v1.1.X        |
-
-**Note:** We have stopped supporting CAPM3 v1alpha4 API version.
+| CAPM3 version | Cluster API version | CAPM3 Release | CAPI Release |
+|---------------|---------------------|---------------|--------------|
+| v1alpha5      | v1alpha4            | v0.5.X        | v0.4.X       | 
+| v1beta1       | v1beta1             | v1.1.X        | v1.1.X       |
+| v1beta1       | v1beta1             | v1.2.X        | v1.2.X       |
 
 ## Deploying the metal3 provider
 
```

Comment:
clusterctl link below is tricky now, because capi has books generated for each branch and most probably we should refer to compatible one too. Book has these versions:

[main.cluster-api.sigs.k8s.io](https://main.cluster-api.sigs.k8s.io/)
[release-1-2.cluster-api.sigs.k8s.io](https://release-1-2.cluster-api.sigs.k8s.io/)
[release-1-1.cluster-api.sigs.k8s.io](https://release-1-1.cluster-api.sigs.k8s.io/)

Reply:
Discussed offline: will be taken care separately